includes/amp-post-template-functions.php

```diff
@@ -15,6 +15,7 @@ function amp_post_template_init_hooks() {
 	add_action( 'amp_post_template_head', 'amp_print_schemaorg_metadata' );
 	add_action( 'amp_post_template_head', 'amp_add_generator_metadata' );
 	add_action( 'amp_post_template_head', 'wp_generator' );
+	add_action( 'amp_post_template_head', 'amp_post_template_add_block_styles' );
```

Comment:
Good idea to do this at this action so that it is added to sites that have customized the `html-start.php` template.